tasks/preflight.yml

```diff
@@ -98,14 +98,13 @@
 - block:
     - name: Get checksum list from github
       set_fact:
-        _checksums: "{{ lookup('url', 'https://github.com/prometheus/node_exporter/releases/download/v' + node_exporter_version + '/sha256sums.txt', wantlist=True) | list }}"
+        _checksums: "{{ lookup('url', 'https://github.com/prometheus/node_exporter/releases/download/v' + node_exporter_version + '/sha256sums.txt') }}"
       run_once: true
 
     - name: "Get checksum for {{ go_arch }} architecture"
       set_fact:
-        node_exporter_checksum: "{{ item.split(' ')[0] }}"
-      with_items: "{{ _checksums }}"
+        node_exporter_checksum: "{{ _checksums | regex_replace('^.*([0-9a-f]{64})[ ]{2}node_exporter-' + node_exporter_version + '.linux-' + go_arch + '[.]tar[.]gz.*$', '\\1') }}"
```

Comment:
I have no idea what is going on here, hence I am really not in favor of merging and maintaining this.

Reply:
https://github.com/cloudalchemy/ansible-node-exporter/issues/174 is causing mucho pain right now.  If this fix does not pass the sniff test, something else would be great.